Source/WebCore/platform/mediastream/gstreamer/GStreamerVideoCapturer.cpp

```diff
@@ -326,7 +326,7 @@ void GStreamerVideoCapturer::reconfigure()
                 selector->maxWidth = *width;
                 selector->maxHeight = *height;
                 selector->maxFrameRate = *frameRate;
-                selector->mimeType = gst_structure_get_name(structure);
+                selector->mimeType = gstStructureGetName(structure).toStringWithoutCopying();
```

Comment:
I think you can't go without copying here because the moment the structure disappears, so does the memory inside the String, and it is kept for later in a struct.